packages/node/src/index.ts

```diff
@@ -514,7 +514,7 @@ export const startDevServer: StartDevServer = async opts => {
     // Got "exit" event from child process
     const [exitCode, signal] = result;
     const reason = signal ? `"${signal}" signal` : `exit code ${exitCode}`;
-    throw new Error(`\`node ${entrypoint}\` failed with ${reason}`);
+    throw new Error(`Function \`${entrypoint}\` failed with ${reason}`);
```

Comment:
Because the function in question here could be a Serverless Function or an Edge Function, I changed the wording of this error. If something else makes more sense, let me know!